frame/support/test/tests/pallet.rs

```diff
@@ -229,9 +229,14 @@ pub mod pallet {
 	}
 
 	#[pallet::error]
+	#[derive(PartialEq, Eq)]
 	pub enum Error<T> {
 		/// doc comment put into metadata
 		InsufficientProposersBalance,
+		Code(u8),
+		#[codec(skip)]
+		Skipped(u128),
```

Comment:
what happens here?

Reply:
If you skip encoding an enum variant, then whatever variant it contains does not contribute to the calculation of the maximum encoded length limit, even when its containing data types exceeds the encoded limit individually, as they are ignored.